static/app/views/starfish/utils/buildEventViewQuery.tsx

```diff
@@ -0,0 +1,69 @@
+import {Location} from 'history';
+
+import {defined} from 'sentry/utils';
+import {ModuleName} from 'sentry/views/starfish/types';
+import {NULL_SPAN_CATEGORY} from 'sentry/views/starfish/views/webServiceView/spanGroupBreakdownContainer';
+
+const SPAN_FILTER_KEYS = [
+  'span.op',
+  'span.domain',
+  'span.action',
+  '!span.module',
+  '!span.category',
+];
```

Comment:
You can use the `SpanMetricsFields`, similar to this refactor https://github.com/getsentry/sentry/pull/52657

Reply:
good call, refactored to use the new constants!